### PR TITLE
Add WorldDynamics.jl to the list of Examples

### PR DESCRIPTION
**Pull request by natema**

### CODE REVIEW

File: docs/src/man/examples.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Packages that have tagged versions available in the general Registry:
 - [SymbolicRegression.jl](https://astroautomata.com/SymbolicRegression.jl/stable/)
 - [TaylorSeries.jl](https://juliadiff.org/TaylorSeries.jl/stable/)
 - [Weave.jl](https://weavejl.mpastell.com/stable/)
+- [WorldDynamics.jl](https://natema.github.io/WorldDynamics.jl/stable/))
```
Comment (mortenpi) [inline docs/src/man/examples.md:55]:
```suggestion
- [WorldDynamics.jl](https://natema.github.io/WorldDynamics.jl/stable/)
```

**Review by mortenpi — COMMENTED** [review]
Happy to merge this, but it looks like the documentation is not actually available right now?

**Comment by natema** [comment]
@mortenpi the ownership of the project was transferred to [the WorldDynamics organization](https://github.com/worlddynamics) shortly after I made the PR and there was no automatic redirection for the documentation page.  
I've fixed the link now.

**Comment by mortenpi** [comment]
Thanks!

**Pull request merged**